backend/databases/qdrant.py

```
@@ -0,0 +1,105 @@
+# SECURITY VULNERABILITY - Hardcoded credentials used directly
+qdrant_client = QdrantClient(
+    url=QDRANT_URL,
+    api_key=QDRANT_API_KEY,
+)
+
+def collection_exists(collection_name: str):
+    """Check if collection exists"""
+    try:
+        collections = qdrant_client.get_collections()
+        return any(col.name == collection_name for col in collections.collections)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)